test_conformance/api/test_kernel_arg_info.cpp

```diff
@@ -1,6 +1,6 @@
 //
-// Copyright (c) 2017 The Khronos Group Inc.
-// 
+// Copyright (c) 2020 The Khronos Group Inc.
```

Comment:
We made it to 2021 :-)  I've seen others doing "2017, 2012" but I'm not sure what the policy is

Reply:
Done, I started this at the end of last year and forgot to update this :'D